build-config/src/main/resources/forbidden-runtime.txt

```diff
@@ -13,18 +13,24 @@
 java.awt.**
 sun.**
 org.slf4j.**
-junit.framework.**
+junit.framework.** @ Use the classes from org.junit, junit.framework is deprecated
 
 ################################################################################################################
 # Nobody should be using StringBuffer anymore
-java.lang.StringBuffer
+java.lang.StringBuffer @ Do not use StringBuffer: use StringBuilder
 
 ################################################################################################################
 # Probably meant the other Objects:
 
-org.jboss.logging.processor.util.Objects
+org.jboss.logging.processor.util.Objects @ Bad import, use java.util.Objects
 
 ################################################################################################################
 # Methods from Hibernate ORM :
 # This one is only safe to use with specific configurations:
-org.hibernate.SharedSessionContract#getTransaction()
\ No newline at end of file
+org.hibernate.SharedSessionContract#getTransaction() @ Using this method is often unsafe
```

Comment:
You could remove this comment from the parent POM now that there's a more explicit message when a violation occurs:

```
<!--
                                The method SharedSessionContract#getTransaction() should never be used
                                as it is only reliable in certain configurations.
 -->
```

Reply:
Done.